tests/scr-readingsystem-support/OPS/content_001.xhtml

```diff
@@ -0,0 +1,58 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>epubReadingSystem object support</title>
+    <script>
+      /**
+       * A simple change that proves that scripting is enabled.
+       * 
+       * Note that the script is more complicated than would be necessary in HTML, due to the fact that the innerHTML method does not
+       * seem to work in xhtml (only in html) :-(
+       */ 
+      function test_ers() {
+        const support = document.getElementById('scripting_support');
+        support.textContent = 'This Reading System supports scripting.';
+
+        try {
+          const ers = navigator.epubReadingSystem;
+          // For code testing
+          // const ers = {
+          //   name : "Test ERS setup",
+          //   version: "0.0.0"
+          // };
+          if (ers !== undefined) {
+            const div_pass = document.getElementById('pass');
+            const div_fail = document.getElementById('fail');
+            const ers_name = document.getElementById('ers_name');
+            const ers_version = document.getElementById('ers_version');
+
+            ers_name.innerText = ers.name;
+            ers_version.innerText = ers.version;
+            div_fail.style.display = 'none';
+            div_pass.style.display = 'block';
+          }
+        } catch (e) {
+          // Let the original, negative display stay as is. 
+        }
+      }
+
+      window.addEventListener('load', test_ers);
+    </script>
+  </head>
+  <body>
+    <p id="scripting_support">This Reading System does not support scripting.</p>
+
+    <div id="fail" style="display: block">
+      <p>The Reading System does not implement the <code>epubReadingSystem</code> object. Test fails.</p>
+    </div>
+
+    <div id="pass" style="display: none">
+      <p>The Reading System implements the <code>epubReadingSystem</code> object, with the following properties:</p>
+      <ul>
+        <li>Reading System Name: <span id="ers_name"></span></li>
+        <li>Reading System Version: <span id="ers_version"></span></li>
```

Comment:
If ERS exists but either or neither of these is populated, does the test still pass? Please clarify.

Reply:
Yep. It is unlikely that an implementation implements the object without the only two attributes that are defined, but let us be complete. I have modified the description by:

> `<p>Test passes, unless one of the two values are set to <code>undefined</code>.</p>`